sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -123,21 +138,35 @@ public class Widget
 }
 ```
 
-### Working with Azure values
+### Work with Azure values
 
 When working with JSON from Azure services, you can learn what properties are available in the JSON response content from the REST API documentation for the service, examples in the protocol method documentation, or by expanding the [Dynamic View](https://learn.microsoft.com/visualstudio/debugger/watch-and-quickwatch-windows) in Visual Studio.
```

Comment:
I think that it would be valuable here to mention that the expected convention for Azure services is that they are case-sensitive and use camelCase names.  It may also be worth mentioning that not all Azure services follow the expected convention and that the REST API documentation is the authority.